packages/lib-react-components/src/Markdownz/Markdownz.spec.js

```diff
@@ -11,4 +13,257 @@ describe('<Markdownz />', function () {
   })
 
   it('renders without crashing', function () { })
+
+  it('renders markdown', function () {
+    expect(wrapper.find('h1')).to.have.lengthOf(1)
+    expect(wrapper.html().includes('Zooniverse')).to.be.true
+  })
+
+  describe('#buildResourceURL', function () {
```

Comment:
Testing the internal implementation of a component like this is brittle. If the internal methods ever change, these tests will need to be rewritten and it will be hard to tell if the new implementation conforms to the original behaviour.

Ideally, the Markdownz tests would compare some input markdown with some output HTML (or JSX?) without really caring about the details of how the markdown is parsed.

Reply:
I see your point of having an overall test of the Markdownz component comparing markdown to outputted jsx for the customizations. The storybook is basically a manual way of doing that, so we just need an automated test too.